### ISSUE TITLE
Milestone 1 Feedback

### ISSUE THREAD

**Issue by zwarham** [UBC-MDS/VanArt#33]
-	Submission (4)
o	4/4 mechanics
-	Section 1 (8 + 2)
o	8/8 reasoning
o	2/2 writing
-	Section 2 (8 + 2)
o	5/8 reasoning
	There was no exploration or discussion into the accuracy of the data and the data source was not cited
o	2/2 writing
-	Section 3 (12 + 2)
o	12/12 reasoning
o	2/2 writing
-	Section 4 (10 + 2 + 3)
o	9/10 visualization
	This is a great visualization however I would work on a little on the color scheme for accessibility and try be more consistent with either picking a sharp edge or smooth curved theme for your sections (instead of putting edges inside curves and vice versa)
o	2/2 writing
o	3/3 reasoning
-	Section 5 (4)
o	4/4 reasoning
-	Code of Conduct (1)
o	1/1 reasoning
-	Contribution Guidelines (1)
o	1/1 reasoning

**Comment by Hongjian-Sam-Li** [comment]
Request to regrade: 
- We want to clarify that for the visualization component, we only needed to provide an initial sketch of our dashboard layout, and it was not a final version. We hope that the color scheme will not result in any point deductions.

- Furthermore, we would like to point out that the instructions for Milestone 1 did not specify that we had to discuss the accuracy of the data or the data source in our reasoning section. We also noted that Florencia stated that we would not be penalized for not implementing everything in our proposal.

We appreciate your feedback on our work, and we believe that with these changes, our proposal will be even better. However, we kindly request that we not lose points for details that were not part of the requirements.

Thank you for your understanding and support.
